service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/RegistryManager.java

```diff
@@ -99,19 +143,84 @@ public static RegistryManager createFromConnectionString(String connectionString
             throw new IllegalArgumentException("RegistryManagerOptions cannot be null for this constructor");
         }
 
-        // Codes_SRS_SERVICE_SDK_JAVA_REGISTRYMANAGER_12_002: [The constructor shall create an IotHubConnectionString object from the given connection string]
-        IotHubConnectionString iotHubConnectionString = IotHubConnectionStringBuilder.createConnectionString(connectionString);
+        this.iotHubConnectionString =
+                IotHubConnectionStringBuilder.createIotHubConnectionString(connectionString);
```

Comment:
Nit: I see some extra spaces here

Reply:
Intellij leaves two tabs when indenting like this, so I don't see them as extra

Reply:
See, I'm not alone. Why does IntelliJ get to set a new standard? :p